book_spec.rb

```diff
@@ -0,0 +1,10 @@
+require './book'
+describe Book do
```

Comment:
- [ ] As it was stated in the requirements(**Add unit tests for all the methods in all the classes:**). We are expected to add unit test for the `add_rental` method for this classes.